app/views/site/help.scala

```diff
@@ -30,6 +30,10 @@ object help:
     row(kbd("space"), trans.playComputerMove()),
     row(kbd("x"), trans.showThreat())
   )
+  private def phonetics(using Lang) = "abcdefgh"
+    .map(_.toString)
+    .map: letter =>
+      frag(s"${letter.capitalize} ", trans.is(), phonetic(letter), ". ")
```

Comment:
this is translation lego and won't work with some languages. Always use variables, never concatenate translated strings.

Reply:
can i change `is` as to `=` or `→`?

Reply:
yeah, good idea